install/kubernetes/helm/istio/charts/kiali/templates/configmap.yaml

```diff
@@ -14,6 +14,8 @@ data:
     server:
       port: 20001
     external_services:
+      istio:
+        url_service_version: http://istio-pilot:8080/version
```

Comment:
seems reasonable.  No user API change.

Do we support installing kiali in a different ns than pilot?

Reply:
Today, no we do not support that (though, with some trickery, I bet you could do it. Kiali is customizable to the point I bet it could go in a different namespace and still work if custom-configured, but we don't really officially support such a deployment). Kiali should be deployed in the same namespace as the rest of the Istio components, including pilot.

Reply:
Maybe `http://istio-pilot.{{ .Release.Namespace }}:8080/version`?

Reply:
@ymesika at this point, it won't matter. If you look at the way the config map is generated, its created in the same namespace as .Release.Namespace which is where Kiali is deployed also. So further qualifying this setting here is superfluous today (it really doesn't help). More would need to be done to get Kiali to work when deployed in a separate namespace.

FWIW: I plan on removing this configmap setting when Istio pulls in the next release of Kiali, since the default will have been fixed and this configmap override will not be necessary. So consider this PR a temporary workaround.